<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1302</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>meiNeumesEditor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>681</width>
      <height>621</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>element</string>
    </property>
    <property name="popupMode">
     <enum>QToolButton::MenuButtonPopup</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>601</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="meta">
     <attribute name="title">
      <string>Meta</string>
     </attribute>
     <widget class="QPushButton" name="metaButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>450</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>create meta data</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="titleLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="titleLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
     <widget class="QLabel" name="composerLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Composer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="composerLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="authorLabel1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
     <widget class="QLabel" name="authorLabel2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
     <widget class="QLabel" name="abailabilityLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Availability</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="authorLine1">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="authorLine2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="commentLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Comments</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="availabilityText">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>240</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="commentsText">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>361</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="languageLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>361</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="languageLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Language</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="keywordsLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>360</y>
        <width>361</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="keywordLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Keywords</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="sources">
     <attribute name="title">
      <string>Sources</string>
     </attribute>
     <widget class="QLineEdit" name="sourceIDLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>90</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sourceAuthorLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sourceTitleLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sourcePubStatusLine">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>331</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sourceIDLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="sourceAuthorLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
     <widget class="QLabel" name="sourceTitleLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
     <widget class="QLabel" name="sourcePubStatusLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Publication Status</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sourceMediumLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>210</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sourceXLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>240</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sourceMediumLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Medium</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sourceYLine">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>240</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="sourceUnitLine">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>240</y>
        <width>31</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sourceDimensionLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Dimension</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>x</string>
      </property>
     </widget>
     <widget class="QLabel" name="sourceUnitLabel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Unit:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sourceConditionLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>270</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sourceConditionLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Condition</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="sourceExtentLine">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>300</y>
        <width>361</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="sourceExtentLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>47</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Extent</string>
      </property>
     </widget>
     <widget class="QLabel" name="sourceHandwritingLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Handwriting Style</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="sourceStyleText">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>330</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="sourceCreateButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>create source</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sourceAddButton">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add source</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sourceFinishButton">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>400</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>finish</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="music">
     <attribute name="title">
      <string>Staff</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>551</height>
       </rect>
      </property>
      <property name="title">
       <string>Staff</string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Linecount</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Clefline</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Clefshape</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Create Staffdef</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>110</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Clef</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Syllable</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>551</height>
       </rect>
      </property>
      <property name="title">
       <string>Syllable</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Page</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Line</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Syllable</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_7">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Create Syllablle</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>210</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Variant</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Comment</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>113</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Pitches</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>591</width>
        <height>551</height>
       </rect>
      </property>
      <property name="title">
       <string>Pitch</string>
      </property>
      <widget class="QLabel" name="label_16">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Pitch</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Pitch</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_14">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Octave</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_18">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Comment</string>
       </property>
      </widget>
      <widget class="QTextEdit" name="textEdit_3">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>130</y>
         <width>113</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_15">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>80</y>
         <width>113</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_19">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="adjustByHand">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>adjust by hand</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1302</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
